### ISSUE TITLE
Mevo+ Bump & Run Issue

### ISSUE THREAD

**Issue by agn507** [springbok/MLM2PRO-GSPro-Connector#123]
I've been using the connector app for a few weeks now and have noticed some possible improvements.

The mevo+ has been hit or miss with a lower type chip. I know many launch monitors struggle with this aspect but the consistent ROI I'm seeing when this happens is a 0.5 VLA. When this happens I'll have a 10-20 foot chip just off the green that goes screaming across the green without every leaving the ground. Could a feature be added in where an exact 0.5VLA could be ignored. 

If there was a possibility of having an ROI that reads if the mevo+ is in chipping vs full swing mode that may help restrict this feature to chipping only. Attached is a screenshot of the ROIs when this happens. I also have a video showing this happening twice in a row from one side of the green then trying to come back.   

![image](https://github.com/user-attachments/assets/8027876e-e2be-41e1-9a22-02c653d6a3b4)

**Comment by wonder99** [comment]
How would we determine if it's in chipping mode?

**Comment by agn507** [comment]
Apologies. Should have included the full window. At the top of the window here you can see the "Full Swing" vs "Chipping".  I'm not too familiar how the OCR works and if it could distinguish between what is selected or not here. Selected is in blue. If we can't I wonder if the guys at flightscope could make it a toggle where it either says "Full Swing" or "Chipping" instead of showing both but having one highlighted. Regardless though I don't think this request is really dependent on what mode you are in. A simple fix would be to just exclude sending any shot that has a 0.5 VLA or -0.5 VLA. 
![image](https://github.com/user-attachments/assets/a0441e6c-7962-4bae-8e57-452157eb71d3)

**Comment by wonder99** [comment]
It's going to be a problem detecting the change in highlight on the swing mode box.  But I'll add the ability to mute shots from Mevo+ that have a VLA of exactly 0.5.  I don't think -0.5 VLA is possible, let me know if you're sure it could happen.

**Comment by wonder99** [comment]
I've submitted code changes to @springbok (#126 ), expect a beta version to test soon.

**Comment by agn507** [comment]
Appreciate it @wonder99 ! If I see the -0.5 VLA and the conditions I'll follow up. I did notice in your commit though that the name may have inadvertently changed 
![image](https://github.com/user-attachments/assets/733ce47e-7e2c-40b4-8871-06e636343c41)

**Comment by wonder99** [comment]
Yes, I let him know about that typo, for him to fix.  Instead, I'll go ahead and fix it, and compile a test image for you

**Comment by wonder99** [comment]
I don't have permission to attach the .exe here.  Can you send me a DM on discord or something?  I'm paul_m0551

**Comment by agn507** [comment]
Thank you! I'm also reaching out to flight scope support about potentially adding an ROI that outputs the current "Hit Type" so it would output Full or Chip. Not sure if that will ultimately be something that could be built on in some cases but worth a try.

**Comment by wonder99** [comment]
The test image has been sent to him for testing

**Comment by wonder99** [comment]
Did my test version of the connector work for you?

**Comment by springbok** [comment]
Latest release with MEVO+ updates https://github.com/springbok/MLM2PRO-GSPro-Connector/releases/tag/V1.04.20
